scripts/w7_git_worktree_bench.py

```
@@ -0,0 +1,788 @@
+        "barriers for the measured operation (ForgeFS file={}/dir={}, "
+        "Git file={}/dir={})".format(
+            forge.get("file", 0),
+            forge.get("dir", 0),
+            git.get("file", 0),
+            git.get("dir", 0),
+        ),
+    )
+
+
+def git_cmd(args, cwd, env=None, check=True):
+    proc = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)